tests/unit/secondsToTimestamp.spec.js

```diff
@@ -0,0 +1,15 @@
+import secondsToTimestamp from '../../src/timestamp';
+
+describe('secondsToTimestamp spec', () => {
+    it('handles positive values', () => {
+        let s = 120;
```

Comment:
You don't have to define a new variable, you can just call the function with this value since it's only used once.